h/static/scripts/widget-controller.js

```diff
@@ -177,7 +177,7 @@ module.exports = function WidgetController(
       // Remove client from list of active search clients
       searchClients.splice(searchClients.indexOf(searchClient), 1);
     });
-    searchClient.get({uri: uri, group: group});
+    searchClient.get({uri: uris, group: group});
```

Comment:
Yes, it looks like Angular (?) converts an array to multiple `uri=` in the query string.